alf/utils/summary_utils.py

```diff
@@ -384,12 +384,14 @@ def __exit__(self, type, value, traceback):
         if self._sync and torch.cuda.is_available():
             torch.cuda.synchronize()
         self._counter['time'] += time.time() - self._t0
-        self._counter['n'] += 1
         if should_record_summaries():
-            alf.summary.scalar(self._tag,
-                               self._counter['time'] / self._counter['n'])
-            self._counter['time'] = .0
-            self._counter['n'] = 0
+            c0 = self._counter['c0']
```

Comment:
This actually records average execution time between two summaries? If the summary interval is very large, then the counter won't be changed for a long time. Is this behavior intended?